alts/worker/tasks.py

```diff
@@ -117,13 +125,16 @@ def is_success(stage_data_: dict):
             else:
                 summary[stage] = {'success': is_success(stage_data)}
         summary['logs'] = runner.uploaded_logs
-        if task_params.get('callback_href'):
-            full_url = urllib.parse.urljoin(CONFIG.bs_host,
-                                            task_params['callback_href'])
-            payload = {'api_version': API_VERSION, 'result': summary}
-            response = requests.post(
-                full_url, json=payload,
-                headers={'Authorization': f'Bearer {CONFIG.bs_token}'})
+    if task_params.get('callback_href'):
```

Comment:
This was in 'finally' section for a reason, when you have failures, so please return it back

Reply:
Done